lib/dor/workflow/client/workflow_routes.rb

```diff
@@ -266,6 +266,14 @@ def workflow(repo: nil, pid:, workflow_name:)
           Workflow::Response::Workflow.new(xml: xml)
         end
 
+        # @param [String] pid id of object
+        # @param [String] workflow_name The name of the workflow
+        # @param [String] process The name of the workflow step
+        # @return [Workflow::Response::Process]
+        def process(pid:, workflow_name:, process:)
```

Comment:
I'm wondering why the consumer isn't doing this?  Do you think adding this might obscure what is going on  (namely "fetch" and "filter")

Reply:
This is a convenience method to surface what one expect to be a basic operation for the workflow client (viz., getting a process) without require code reading / understanding.

See `workflow_status` as another similar convenience method provided in the client.

Reply:
But I'm not going to fall on my sword over this one if you feel strongly.

Reply:
Not really.